config/resources/master-jobs-domain.lisp

```diff
@@ -0,0 +1,68 @@
+(define-resource job ()
+  :class (s-prefix "cogs:Job")
```

Comment:
aren't these missing in the mu-auth config?

Reply:
Yes, probably missing. The problem is that I can not test this yet without any data/reports. I simply don't know what else I am missing here.